docs/How_To/Slack Bot/2_run_slackbot.rst

```diff
@@ -86,15 +86,22 @@ Run with Virtual Environment
 
        conda activate slackbot
 
-5. Install the dependencies of the slackbote: 
+5. Install the dependencies of the slackbot: 
    
    * Install dependency with `Poetry <https://python-poetry.org/>`__ (recommended, this will give you the ability to edit the code in `sherpa_ai` without rebuild) 
 
       .. code:: bash
 
          cd src/apps/slackapp
          poetry install
-      
+
+     Followed by:
```

Comment:
Maybe it makes more sense to install the sherpa_ai dependencies first and then `poetry install`  for the slack bot?

Reply:
Thanks Percy. Although it would work either way, the order you suggest makes much more logical sense. I have made the change. Please let me know what you think.